akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala

```diff
@@ -396,16 +396,20 @@ private[remote] abstract class InboundCompression[T >: Null](
       case None ⇒
         inboundContext.association(originUid) match {
           case OptionVal.Some(association) ⇒
-            if (alive) {
+            if (association.associationState.isQuarantined(originUid)) {
+              // FIXME cleanup compresssion for quarantined associations, see #23967
+              log.debug("Ignoring {} for quarantined originUid [{}].", Logging.simpleName(tables.activeTable), originUid)
```

Comment:
Do we really want to have these just as `DEBUG` messages? Could be an indicator that something is quite wrong if we receive lots of those.

Reply:
I have changed this in PR https://github.com/akka/akka/pull/24023, which is on top of this one.